src/main/java/io/valkey/CommandObjects.java

```diff
@@ -2417,7 +2417,7 @@ public final CommandObject<Long> geosearchStoreStoreDist(byte[] dest, byte[] src
   }
   // Geo commands
 
-  // Hyper Log Log commands
```

Comment:
[HyperLogLog](https://valkey.io/topics/hyperloglogs/) is a redis structure, do not need modify.